drivers/include/lsm6ds3.h

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    driver_LSM6DS3 LSM6DS3 3D accelerometer/gyroscope
+ * @ingroup     drivers
+ * @brief       Device driver for the LSM6DS3 3D accelerometer/gyroscope
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for the LSM6DS3 3D accelerometer/gyroscope
+ *
+ * @author      René Herthel <rene-herthel@outlook.de>
+ */
+
+#ifndef LSM6DS3_H
+#define LSM6DS3_H
+
+#include <stdint.h>
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief TODO: The sensors default I2C address
+ */
+#define LSM6DS3_ACC_DEFAULT_ADDRESS   (0x00)
+#define LSM6DS3_GYR_DEFAULT_ADDRESS   (0x00)
```

Comment:
Ok. Then the next question is, do you need these defines? Or as you want to have them in for "template"-reasons, you could add a comment...